src/app/home/home.component.ts

```diff
@@ -16,11 +16,12 @@ export class HomeComponent implements AfterViewInit {
   constructor(public loader: NgxSmartLoaderService) {
   }
 
-  ngAfterViewInit() {
-    // For demo auto start purposes
-    setTimeout(() => {
+  ngOnInit(): void {
       this.loader.start('myLoader');
-    }, 500);
+      setTimeout(() => {
+        console.log('HomeComponent\'s loader auto hide');
```

Comment:
I guess this console.log is not a mistake. But let me highlight it, just in case.